features/steps/partcad-cli/commands/init.py

```
@@ -0,0 +1,134 @@
+
+
+@given('I am in "{directory}" directory')
+def step_impl(context: Context, directory):
+    # TODO: @alexanderilyin: use dedicated object for test directories
+    # Create a temporary directory for the test
+    os.makedirs(directory, exist_ok=True)
+    context.test_dir = tempfile.mkdtemp(
+        prefix="partcad-cli-",
+        dir=directory,
+    )
+    logging.debug(f"Created temporary workspace: {context.test_dir}")
```

Ruff (0.8.0)

Logging statement uses f-string

(G004)